react/ConditionLayoutProduct.tsx

```diff
@@ -29,6 +29,9 @@ export const PRODUCT_SUBJECTS = {
 const Product: StorefrontFunctionComponent = ({ children }) => {
   const { product, selectedItem } = useProduct() as any
 
+  // Sometimes it takes a while for useProduct() to return the correct results
+  if ( product == null || selectedItem == null) return null
```

Comment:
```suggestion
  if (product == null || selectedItem == null) return null
```

Reply:
Is lint properly set in this project?

Reply:
Can we have a `product` defined but a `null` `selectedItem` in a valid scenario?